custom/conf/app.example.ini

```diff
@@ -1533,7 +1533,7 @@ PATH =
 ;SENDMAIL_PATH = sendmail
 ;;
 ;; Specify any extra sendmail arguments
-;SENDMAIL_ARGS =
+;SENDMAIL_ARGS = ; WARNING: if your sendmail program interprets options you should set this to "--" or terminate these args with "--"
```

Comment:
Would move this to the comment 1 line above.

Reply:
it's not the default.

Reply:
Ok, but comment should still be moved.